modules/mono/build_scripts/build_assemblies.py

```diff
@@ -342,6 +342,13 @@ def build_all(msbuild_tool, module_dir, output_dir, godot_platform, dev_debug, p
 def main():
     import argparse
     import sys
+    import os.path
+
+    if not os.path.exists("modules/mono/glue/GodotSharp/GodotSharp/Generated/"):
+        print(
+            "WARNING: Mono glue not found.\nConsider running the command `<godot_binary> --generate-mono-glue ./modules/mono/glue` from the Godot source root and recompiling using `mono_glue=yes`.\n",
```

Comment:
The `mono_glue` option does no longer exist in 4.0 and there's no need to build the engine twice anymore.

Reply:
Good to know. In that case, we should get a warning when trying to use it, and the online docs need to be updated.
I'll update this PR to remove this part.